shiny-tic-toc.js

```diff
@@ -148,14 +166,26 @@ function downloadCsvFile(data) {
 
 function getMeasurements() {
   return performance.getEntries()
-    .filter(entry => entry.entryType === 'measure');
+    .filter(entry => entry.entryType === 'measure')
+    .map(entry => ({
+      name: entry.name,
+      duration: entry.duration,
+      startTime: entry.startTime,
+      type: entry.detail.type
+    }));
+
 }
 
 function prepareCsvData() {
-  const dataHeader = ["measurement_id", "duration (ms)", "start_time"];
+  const dataHeader = ["measurement_id", "duration (ms)", "start_time", "type"];
```

Comment:
```suggestion
  const dataHeader = ["measurement_id", "type", "start_time", "duration (ms)"];
```

Such an order would feel more natural to me, WDYT?

Reply:
Good feedback! I agree!

Reply:
Addressed in e52f503fdc95a3b51c740d0a65bb598632c3f5dd